### PR TITLE
exporters: migrate remaining sensitive config fields to configopaque.String

### PR DESCRIPTION
**Pull request by gbbr**
This change migrates the remaining exporters from #17273 to use configopaque.String:

- [x] [exporter/alibabacloudlogservice] Use configopaque for access_key_secret field
- [x] [exporter/azuredataexplorer] Use configopaque for application_key field
- [x] [exporter/azuremonitor] Use configopaque for instrumentation_key field
- [x] [exporter/coralogix] Use configopaque for private_key field
- [x] [exporter/elasticsearch] Use configopaque for api_key and password fields
- [x] [exporter/influxdb] Use configopaque for token and password fields
- [x] [exporter/instana] Use configopaque for agent_key field
- [x] [exporter/logicmonitor] Use configopaque for apitoken::access_key fields
- [x] [exporter/logzio] Use configopaque for account_token field
- [x] [exporter/mezmo] Use configopaque for ingest_key field
- [x] [exporter/pulsar] Use configopaque for auth::Token::Token and auth::athenz::private_key fields
- [x] [exporter/sapm] Use configopaque for access_token field
- [x] [exporter/tencentcloudlogservice] Use configopaque for secret_key field

### CODE REVIEW

File: .chloggen/gbbr_opaque_exporters-azuredataexplorer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```
Comment (MovieStoreGuy) [inline .chloggen/gbbr_opaque_exporters-azuredataexplorer.yaml:2]: Can I ask how is this breaking?

**Review by MovieStoreGuy — CHANGES_REQUESTED** [review]
I approve of the changes however, I don't believe we need changelogs for these since the changes are non impacting for the users .

File: .chloggen/gbbr_opaque_exporters-azuredataexplorer.yaml (hunk 1 above)
Comment (gbbr) [reply .chloggen/gbbr_opaque_exporters-azuredataexplorer.yaml:2]: I somewhat agree with you. Can you take this discussion to the issue please? That's where it was described as a breaking change, so we should have the discussion scoped to all of the changes in this category.

Some PRs were already merged like this btw.

**Review by MovieStoreGuy — APPROVED** [review]
Spoken with @mx-psi and I agree that this is worth the change logs

**Review by codeboten — APPROVED** [review]

**Comment by gbbr** [comment]
Thanks for having the chat with Pablo, Sean.

**Pull request merged**